packages/shared/forks/ReactFeatureFlags.test-renderer.native-fb.js

```diff
@@ -69,7 +69,7 @@ export const enableTaint = true;
 export const enableTransitionTracing = false;
 export const enableTrustedTypesIntegration = false;
 export const enableUpdaterTracking = false;
-export const enableUseDeferredValueInitialArg = __EXPERIMENTAL__;
+export const enableUseDeferredValueInitialArg = false;
```

Comment:
I'm not sure why this got set to `experimental` in https://github.com/facebook/react/pull/29903  but seems like it would be pretty safe to switch to `true` seeing as that is the value for native-fb and native-oss now. cc @yungsters